apps/omg/test/omg/state/transaction/fee_test.exs

```diff
@@ -0,0 +1,94 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.State.Transaction.FeeTest do
+  @moduledoc false
+
+  use ExUnitFixtures
+  use ExUnit.Case, async: true
+
+  alias OMG.Output
+  alias OMG.State.Transaction
+
+  @eth OMG.Eth.RootChain.eth_pseudo_address()
+  @other_token <<127::160>>
+
+  describe "new/2" do
+    @tag fixtures: [:alice]
```

Comment:
can we step back and stop using fixtures for new features and code? cc @achiurizo @unnawut

Reply:
Absolutely :exclamation:  Thanks for spotting this :+1: